src/reboost/build_hit.py

```diff
@@ -245,6 +245,9 @@ def build_hit(
 
         # loop over processing groups
         for group_idx, proc_group in enumerate(config["processing_groups"]):
+            msg = f"... starting group {proc_group.get('name', 'default')}"
+            log.info(msg)
+
             proc_name = proc_group.get("name", "default")
```

Comment:
```suggestion
            msg = f"... starting group {proc_group.get('name', 'default')}"
            log.info(msg)

            proc_name = proc_group.get("name", "default")
```

you can avoid the repeated call to `.get()`